### DISCUSSION TITLE
Authentication via OIDC?

### DISCUSSION THREAD

**Discussion by manuschillerdev** [verdaccio/verdaccio#2551] [Ideas]
Hi,

I am looking for a possibility to integrate Verdaccio in our existing infrastructure via OIDC (in our case: Keycloak, oauth2).
I was looking at the current plugin documentation (https://verdaccio.org/docs/plugin-auth) 

Is something like this already possible?
It would be cool, if we could use existing groups for different authorization scopes in verdaccio as well.

I don't have any more concrete suggestions, so I'm looking forward to any hints in the right direction :)

**Comment by juanpicado** [comment]
hi @manuschillerdev I've never tried myself, I've seen some initiatives but I ignore the progress. I think you have already found one.
